composer.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "silverstripe/login-forms",
+    "description": "A collection of templates for the CMS login screens",
+    "type": "silverstripe-vendormodule",
+    "license": "BSD-3-Clause",
+    "homepage": "https://github.com/silverstripe/login-forms",
+    "keywords": [
+        "silverstripe",
+        "template",
+        "login",
+        "styling"
+    ],
+    "require": {
+        "silverstripe/framework": "^4"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^5.7",
```

Comment:
Please add `"roave/security-advisories": "dev-master"`?

Reply:
Yep, this is a PHPStorm/EA Inspections suggestion which we haven't standardised on yet

Reply:
Sorry I misread this, same comment for this as the other - please don’t add anything that isn’t part of our standardised set already without an RFC and widespread adoption